### DISCUSSION TITLE
Mini 11 as a Dash Cam?

### DISCUSSION THREAD

**Discussion by AGuetz** [gopro/labs#712] [General]
So I followed the manual firmware steps, copied your QR code but doesn't work on my HERO11 Mini. I have my camera connected to its dedicated usb power source and it boots up but does not begin/stop recording upon car movement. Your video shows it working on your 11 mini so wondering if it could be something wrong with my camera and worth returning?

**Comment by dnewman-gpsw** [comment]
No, it's not your camera. It's either the power supply is not fully rated correctly or the script is no longer compatible with the current firmware. There are several threads discussing this very issue on this discussion forum. There are alternate scripts for you to try.

**Reply by AGuetz** [reply]
Could you please provide me with those alternate scripts or point me in the right direction to locate them?

**Reply by dnewman-gpsw** [reply]
Check out the threads on USB power triggers.

**Reply by dnewman-gpsw** [reply]
Directly from https://github.com/gopro/labs/discussions/320 (only two post ago)

This works, it adds a limit that the battery must be 80% full, but you get the idea

![DashCam _ 80%-1_8_2024, 14_17_42](https://github.com/gopro/labs/assets/10856084/eb553e09-146e-411f-87ac-c9bf727d7d61)

**Reply by AGuetz** [reply]
Ok great. I will give it a shot. Thanks for your help!

**Reply by eneerhut** [reply]
How did it go @AGuetz?

**Comment by AGuetz** [comment]
Not as I had hoped for. On a youtube video that David Newman posted, he shows his Mini 11 turn on and begin recording when it recognizes his car is in motion and turning off when the camera is stationary for 20 seconds or so. Furthermore in his video he states that the custom QR code he created also allows the memory card to overwrite itself when it gets full.

 

The only custom QR code I was able to get to work was one that turns on and begins to record when I have the usb cable plugged in and stops recording when I unplug it. Again, not exactly what I was hoping for but doable I guess. Any insight/thoughts?

**Reply by eneerhut** [reply]
Thanks for the update. I'm just starting to experiment with this myself. My use case was similar to a dashcam use case, but I am interested in using the GoPro for [street-level imagery collection](https://www.mapillary.com/app/user/eneerhut?lat=37.713550549541&lng=-119.73167709004&z=19.9&pKey=384410840737309&focus=photo&x=0.04190439621583625&y=0.5592638315632026&zoom=0).

So far the GPS start stop works very nicely for me using the GoPro Max, but I have not tried using power to start/stop capture.

**Comment by AGuetz** [comment]
Got it. Yeah, I think the max has the built in GPS while the mini does not and therefore probably the reason the start/stop feature isn’t working on mine.

 

Best of luck!

**Reply by eneerhut** [reply]
Yeah exactly. Shame some of the newer cameras drop GPS, but I guess that was a cost decision.
Good luck to you too!
